expression/expressions/helper.go

```diff
@@ -467,3 +475,45 @@ func EvalBoolExpr(ctx context.Context, expr expression.Expression, m map[interfa
 
 	return x != 0, nil
 }
+
+// CheckOneColumn checks whether expression e has only one column for the evaluation result.
+// Now most of the expressions have one column except Row expression.
+func CheckOneColumn(e expression.Expression) error {
+	_, ok := e.(*Row)
```

Comment:
Not need to check if len(e.values) == 1?
Or why not rename this function to "IsRowExpr" and return bool?